grapple/actions.py

```diff
@@ -23,7 +23,7 @@
 from .types.images import ImageObjectType
 from .types.pages import PageInterface, Page
 from .types.streamfield import generate_streamfield_union
-from .helpers import streamfield_types
+from .helpers import streamfield_types, fields_middlewares
```

Comment:
```suggestion
from .helpers import streamfield_types, field_middlewares
```

it is a nitpick. `field_middlwares` reads easier than `fields_middlewares`